index.php

```diff
@@ -15,18 +15,29 @@
     exit();
 }
 
-$db = false;
-if ($config['db']['connect']) {
-    $db = connectToDB($config['db']);
-}
+$installed_directory = __DIR__;
```

Comment:
installed_directory is a bad name.

What about "installation_directionry ? current_directory eg